source/_patterns/01-molecules/article-type/article-type.yaml

```diff
@@ -0,0 +1,3 @@
+items:
+  - label: Type of article
+    value: Research Article
```

Comment:
Should probably match https://github.com/libero/pattern-library/blob/085a95e658c89acb0a8f31b4b4601f2af9e9a9b6/source/_patterns/01-molecules/tag-list/tag-list.yaml#L3-L6 and so https://github.com/libero/pattern-library/blob/085a95e658c89acb0a8f31b4b4601f2af9e9a9b6/source/_patterns/01-molecules/tag-list/tag-list.twig#L41-L45 too.

The label can then be an `aria-label`.

Reply:
Have changed to matched other two yaml files and added aria-label